artemis-quorum-ri/src/test/java/org/apache/activemq/artemis/quorum/zookeeper/CuratorDistributedLockTest.java

```diff
@@ -84,8 +82,6 @@ public void setupEnv() throws Throwable {
       }
       testingServer = new TestingCluster(clusterSpecs);
       testingServer.start();
-      // start waits for quorumPeer!=null but not that it has started...
-      Wait.waitFor(this::ensembleHasLeader);
```

Comment:
Did this actually change on the Curator side, or did you just remove the 'wait for start' as the getQuorumPeer method went away?

Reply:
`ensembleHasLeader` used 'getQuorumPeer' inside

Reply:
I realise, which is why I mentioned it in my question. What I wondered is whether the reason it did so, to wait for _actual startup_, has changed on the curator side. If it didn't then you have just removed a step that I expect was there for a reason.

Reply:
reverted